features/steps/validation_handling.py

```diff
@@ -207,12 +225,15 @@ def wrapped_step(func):
 
 def execute_step(fn):
     is_global_rule = False
+    is_full_stack_rule = False
     while hasattr(fn, '__wrapped__'): # unwrap the function if it is wrapped by a decorator in casse of catching multiple string platterns
```

Comment:
```suggestion
    while hasattr(fn, '__wrapped__'): # unwrap the function if it is wrapped by a decorator in case of catching multiple string platterns
```